phpBB/phpbb/language/language.php

```diff
@@ -0,0 +1,571 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\language;
+
+use phpbb\language\exception\invalid_plural_rule_exception;
+
+/**
+ * Wrapper class for loading translations
+ */
+class language
+{
+	/**
+	 * Global fallback language
+	 *
+	 * ISO code of the language to fallback to when the specified language entries
+	 * cannot be found.
+	 *
+	 * @var string
+	 */
+	const FALLBACK_LANGUAGE = 'en';
+
+	/**
+	 * @var array	List of common language files
+	 */
+	protected $common_language_files;
+
+	/**
+	 * @var bool
+	 */
+	protected $common_language_files_loaded;
+
+	/**
+	 * @var string	ISO code of the default board language
+	 */
+	protected $default_language;
+
+	/**
+	 * @var array	Language fallback array (the order is important)
+	 */
+	protected $language_fallback;
+
+	/**
+	 * @var array	Array of language variables
+	 */
+	protected $lang;
+
+	/**
+	 * @var array	Loaded language sets
+	 */
+	protected $loaded_language_sets;
+
+	/**
+	 * @var \phpbb\language\language_file_loader Language file loader
+	 */
+	protected $loader;
+
+	/**
+	 * @var string	ISO code of the User's language
+	 */
+	protected $user_language;
```

Comment:
fixed (github doesn't show it as an outdated diff for some reason, at least not at my end)